custom_module/cypher_queries/db_management_queries.py

```diff
@@ -0,0 +1,17 @@
+from promg import Query
+
+class DBManagementQueries:
+    @staticmethod
+    def create_db(dbname):
```

Comment:
We already have this query available in get_clear_db_query(db_name) in db_management_ql.py in PromG.

So, maybe better to do the following in db_management_ql.py?
```suggestion
    def get_create_db_query(dbname):
    	query_str = '''
            CREATE OR REPLACE DATABASE $db_name
            WAIT
        '''

        return Query(query_str=query_str, database="system", template_string_parameters={"db_name": db_name})
       
      def get_clear_db_query(dbname):
       	return get_create_db_query(dbname)
```